Plots/Trajectories/NCL_traj_1.py

```diff
@@ -0,0 +1,86 @@
+"""
```

Comment:
Why is this file, NCL_traj_1.py, being included in this PR? Are you branching from master, or from another branch? Should be the former.